src/officehours_api/backends/bluejeans.py

```diff
@@ -193,6 +194,7 @@ def get_public_data(cls) -> BackendDict:
             'friendly_name': cls.friendly_name,
             'enabled': cls.enabled,
             'docs_url': cls.docs_url,
+            'profile_url': cls.profile_url,
             'telephone_num': cls.telephone_num,
             'intl_telephone_url': cls.intl_telephone_url
         }
```

Comment:
@jlost, this method is identical across the three backends, which begs the question whether it should be inherited from a parent class. I introduced this method, but I'm fairly certain you've had occasion in the past to consider something like this (and thus that you have some opinion about this). Any thoughts you'd like to share?

Reply:
My thoughts on this aren't very nuanced; I think a base class makes sense to share methods like this and also show which method signatures need to exist in all `Backend`s.

Reply:
Cool, this isn't directly related to the issue being addressed here, so I've created a separate one: https://github.com/tl-its-umich-edu/remote-office-hours-queue/issues/311